trin-core/src/jsonrpc/endpoints.rs

```diff
@@ -65,6 +66,9 @@ impl FromStr for TrinEndpoint {
             "discv5_routingTableInfo" => Ok(TrinEndpoint::Discv5Endpoint(
                 Discv5Endpoint::RoutingTableInfo,
             )),
+            "portal_historyRoutingTableInfo" => Ok(TrinEndpoint::HistoryEndpoint(
+                HistoryEndpoint::RoutingTableInfo,
+            )),
```

Comment:
I'd go for isolating this endpoint & its logic into a separate pr and add a test for it to `ethportal-peertest`. As well as support in the state network. More of a nitpick for now, since we'll get tests for all the endpoints sooner or later. And it's probably a bit too late for this pr, but down the road easily extractable chunks of logic like this are better off in their own pr to reduce clutter.